word_language_model/generate.py

```diff
@@ -51,14 +51,22 @@
 ntokens = len(corpus.dictionary)
 hidden = model.init_hidden(1)
 input = torch.randint(ntokens, (1, 1), dtype=torch.long).to(device)
+has_mask = False
```

Comment:
You can just pass this in directly, since it's not an argument. If you want to document it more, you can pass it as a keyword argument.